web_widget_uom/models/uom_uom.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+
+from odoo import fields, models
+
+
+class UoM(models.Model):
+    _inherit = "uom.uom"
+
+    decimal_places = fields.Integer(
+        string="Decimal Places",
+        default=2,
+    )
+
+    _sql_constraints = [
+        (
+            "uom_decimal_places_positive",
+            "CHECK(decimal_places >= 0)",
+            "Decimal places must be strictly bigger or equal than one",
```

Comment:
```suggestion
            "Decimal places must be strictly bigger or equal than zero",
```
as per CHECK(..) above

Reply:
@ajaniszewska-dev u r right 😬 fixed